docs/7_OngoingWork.md

```diff
@@ -0,0 +1,328 @@
+# Transfer learning and fine-tuning
```

Comment:
Main title doesn't match the file name ... The displayed name in Doxygen won't be the one expected.
The main title should be `Ongoing work` and this title should be h2